routers/api/v1/api.go

```diff
@@ -32,6 +33,9 @@
 //     SecurityDefinitions:
 //     BasicAuth:
 //          type: basic
+//     ReverseProxyAuth:
+//          type: reverseProxy
+//          description: Reverse proxy auth using HTTP header.
```

Comment:
This is incorrect but as above I don't think we should be offering this in our swagger.

```suggestion
```

If you really wanted to put this in it would need to be something like:

```suggestion
//     ReverseProxyAuth:
//          type: apiKey
//          in: header
//          name: {{ReverseProxyAuthUser}} 
//          description: Reverse proxy auth using HTTP header.
```

and put something in to the swagger renderer that showed which header was the important one.

Reply:
ReverseProxyAuth removed from swagger as suggested.